packages/map/lib/src/leaflet/components/controls/leaflet-draw-control/leaflet-draw-control.component.ts

```diff
@@ -1,11 +1,51 @@
 import { Component } from '@angular/core';
 
-import { LeafletMap } from '../../../classes/leaflet-map';
+import { LeafletMap, MODES } from '../../../classes/leaflet-map';
+import { LeafletDrawOptions } from '../../../types/leaflet.types';
 
 @Component({
 	selector: 'aui-leaflet-draw-control',
 	templateUrl: './leaflet-draw-control.component.html',
 })
 export class LeafletDrawControlComponent {
 	map: LeafletMap;
+	modes = MODES;
+	baseOptions: LeafletDrawOptions = {
```

Comment:
Technically correct, but I would much rather see these configuration options be served from a service that is DI in the component. Keep the component DUMB.